README.md

```diff
@@ -23,8 +23,9 @@ The code contains seven sample backends:
 * p4c-dpdk: can be used to target the DPDK software switch (SWX) pipeline
   https://doc.dpdk.org/guides/rel_notes/release_20_11.html,
 * p4c-ebpf: can be used to generate C code which can be compiled to [eBPF](https://en.wikipedia.org/wiki/Berkeley_Packet_Filter)
-  and then loaded in the Linux kernel. The eBPF backend currently implements two architecture models:
-  [ebpf_model.p4 for packet filtering](./backends/ebpf/README.md) and [the fully-featured PSA (Portable Switch Architecture) model](./backends/ebpf/psa/README.md).
+  and then loaded in the Linux kernel. The eBPF backend currently implements three architecture models:
+  [ebpf_model.p4 for packet filtering, xdp_model.p4 for XDP](./backends/ebpf/README.md) and
```

Comment:
`xdp_model.p4`, which is an extension, is not part of the compiler yet. 
https://github.com/vmware-archive/p4c-xdp

It is sensible to merge it in at this point.

Reply:
As above, it's supported now. The [`xdp_model.p4`](https://github.com/p4lang/p4c/blob/main/p4include/xdp_model.p4) file is part of this repo, and the eBPF backend autodetects to target here:

https://github.com/p4lang/p4c/blob/main/backends/ebpf/ebpfProgram.cpp#L33C37-L55

Reply:
Uff, I completely missed that, even though I even commented on the PR. I guess because I checked for the `xdp_target` and extension instead. https://github.com/vmware-archive/p4c-xdp/blob/master/xdp_target.py

Reply:
It would be nice if we could also port all the existing tests from the old repository. But this we can do in a different PR.

Reply:
Can we create an issue to port the old XDP tests to the compiler, so that we do not forget to  track this?

Reply:
Done: #4337